src/main/java/ConfirmBuyPropertyUseCase/ConfirmBuyPropertyInteractor.java

```diff
@@ -0,0 +1,97 @@
+package ConfirmBuyPropertyUseCase;
+
+import MainEntities.CampaignAccess;
+import MainEntities.Player;
+import Properties.Property;
+import Tiles.PropertyTile;
+
+public class ConfirmBuyPropertyInteractor implements ConfirmBuyPropertyInputBoundary {
+
+    private final ConfirmBuyPropertyOutputBoundary result;
+    private final Property property;
+    private final Player player;
+    private CampaignAccess campaignAccess;
+
+
+    public ConfirmBuyPropertyInteractor(ConfirmBuyPropertyOutputBoundary buyPropertyOutputBoundary,
+                                        CampaignAccess campaignAccess, Player player, Property property) {
+        PropertyTile propertyTile;
+        propertyTile = (PropertyTile) campaignAccess.getCampaign().getTileUnderPlayer(player);
+        this.result = buyPropertyOutputBoundary;
+        this.player = campaignAccess.getCampaign().getCurrentPlayer();
+        this.property = propertyTile.getProperty();
+
+    }
+
+    public CampaignAccess getCampaignAccess() {
+        return campaignAccess;
+    }
+
+    public void setCampaignAccess(CampaignAccess campaignAccess) {
+        this.campaignAccess = campaignAccess;
+    }
+
+    /**
+     *
+     * @param decision The decision of the player rather to purchase or not purchase
+     *                 the landed on property. If decision is true, then the player attempts
+     *                 to purchase the property with buyProperty().
+     */
+
+    public void decisionBuyProperty(boolean decision){
+        if (decision){
+            buyProperty(campaignAccess);
+        }
+    }
+
+    /**
+     *
+     * @param  campaignAccess Accesses the Campaign class.
+     * @return An integer message informing the player about purchasing the property. There are 3
+     *         types of messages indicated as 0, 1, and 2. If message is equals to 0, the property
+     *         is already purchased and has an owner; if the message is equals to 2, the player does
+     *         not have enough funds to purchase this property; if the message is equals to 1, the
+     *         player has enough funds and thus can purchase this property, when player attempts to
+     *         buy the property, player successfully purchases the property.
+     */
+
+    public int buyProperty(CampaignAccess campaignAccess){
```

Comment:
I don't think you need campaignAccess as a parameter since it's already initialized in the constructor.